README.md

```diff
@@ -1 +1,37 @@
 # workbench
+
+## Getting the code
+
+* Run "git clone https://https://github.com/all-of-us/workbench"
+
+To make changes, do:
+
+* git checkout master
+* git pull
+* git checkout -b USERNAME/BRANCH_NAME
+* (make changes and git add / commit them)
+* git push -u origin USERNAME/BRANCH_NAME
+
+And make a pull request in your browser at 
+https://github.com/all-of-us/workbench based on your upload.
+
+After responding to changes, merge in GitHub.
+
+
+## Setup
+
+* Download Gradle 4.0 from https://services.gradle.org/distributions/gradle-4.0-bin.zip
+
+* Unzip it and add bin/gradle to the path.
+
+* Run tools/setup_env.sh.
```

Comment:
I have mixed feelings about that. Shell scripts are good for doing some things simply (e.g. running command lines), Python for more complex things. For something like "set up my local environment" which is just going to consist of running command lines, shell seems OK to me.

Also given that we're programming in Java, there's no particular reason at this point to require somebody developing workbench to have Python installed. (Not that it would be a big deal to require it at some stage.)

Another option is to use Gradle for this sort of thing. Let me know if you prefer that.